ta/avb/entry.c

```diff
@@ -0,0 +1,258 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
C++-style comment for the SPDX line in a `.c` file

Reply:
I'll fix